test_scripts/Capabilities/PersistingHMICapabilities/031_Suspending_of_RAI_request_processing_in_case_cash_file_does_not_exist_no_cap_response.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0249-Persisting-HMI-Capabilities-specific-to-headunit.md
+--
+-- Check that SDL suspend of RAI request processing from mobile app in case HMI does not send all HMI Capabilities
+--  (VR/TTS/RC/UI/Buttons.GetCapabilities/,VR/TTS/UI.GetSupportedLanguages/GetLanguage, VehicleInfo.GetVehicleType)
+--  response due to timeout. HMI capabilities cache file (hmi_capabilities_cache.json) doesn't exist
+--  on file system and ccpu_version matches with received ccpu_version from HMI
+--
+-- Preconditions:
+-- 1  Value of HMICapabilitiesCacheFile parameter is defined (hmi_capabilities_cache.json) in smartDeviceLink.ini file
+-- 2. HMI capabilities cache file doesn't exist on file system
+-- 3. SDL and HMI are started
+-- 4. HMI sends all HMI capabilities (VR/TTS/RC/UI etc) to SDL
```

Comment:
@VjKlepikov Seems this precondition step is redundant. I would propose to remove it.